setup.md

```diff
@@ -211,31 +211,3 @@ If you want to reverse or “undo” the changes made by `conda init`, then you
 `conda init` command and pass the `--reverse` option. Again, in order for the reversal to take 
 effect you will likely need to close and restart your shell session.
 
-## Workspace Setup
```

Comment:
Why remove this from the `setup.md` file? Typically SWC and DC lessons will include the creation of any working directory for the lesson at part of the `setup.md` file.

Reply:
Mostly because  I found this section a bit hidden; for instance if using binder, we would probably skip this step. And we do not really need a project folder before this episode 2 (episode 1 is mostly about the context/introduction).

But I agree we could move it either at the bottom of episode 1 or at the top of episode 2. What do you think?

Reply:
I moved it on top of episode 2. Alternatively, we could move it back to setup but add a table of content (so that everyone understands what steps are required.

Reply:
It is important that the section be included in the `setup.md` file. We can also add it in the lessons at the top of episode 2 if you feel it needs the extra visibility.